### DISCUSSION TITLE
Issue #6290 went 404

### DISCUSSION THREAD

**Discussion by mitsukuri** [microsoft/vscode-cpptools#8253] [Q&A] [answered]
There was an issue **#6290** which tracked the status of c++20 module support. Trying to check that today, I'm getting error 404.

What happened to the issue and how do we track the progress on that?

**Comment by bobbrow** [comment, marked as answer]
Someone deleted it instead of closing it (not sure who).  This is the issue to follow now: #6302

**Reply by mitsukuri** [reply]
Oh, thanks for the heads-up!
